pkg/operator/ceph/disruption/clusterdisruption/pools.go

```diff
@@ -117,7 +117,7 @@ func (r *ReconcileClusterDisruption) reconcileCephObjectStore(cephObjectStoreLis
 
 		rgwCount := objectStore.Spec.Gateway.Instances
 		minAvailable := &intstr.IntOrString{IntVal: rgwCount - 1}
-		if minAvailable.IntVal <= 1 {
+		if minAvailable.IntVal < 1 {
```

Comment:
Can you give more rationales for this change? Isn't this `== 0` somehow too?

Reply:
My intention is to create PDB of RGW if ObjectStore's `instances` field is >= 2. However, `minAvailable.IntVal` becomes 1 and then the creation of PDB is skipped. In other words, in the current implementation, at least three desired RGW instances are needed to create PDB. I removed `=` to prevent this problem.

On the other hand, if `instances` is 1, we can safely skip the creation of PDB because it's useless.

Reply:
Thanks please add this to your commit message.

Reply:
Thanks, fixed.